demo/package.json

```diff
@@ -4,21 +4,24 @@
   "version": "0.0.0",
   "type": "module",
   "scripts": {
-    "start": "vite"
+    "clean": "tsc -b -c",
+    "compile": "tsc",
+    "start": "vite",
+    "start:debug": "vite --debug --force"
   },
   "devDependencies": {
-    "@types/throttle-debounce": "^5.0.0",
-    "typescript": "^4.7.4",
-    "vite": "^3.0.9"
+    "@types/throttle-debounce": "~5.0.0",
+    "typescript": "~4.9.5",
+    "vite": "~4.1.4"
   },
   "dependencies": {
-    "monaco-editor": "~0.34.0",
-    "throttle-debounce": "^5.0.0",
-    "vscode": "npm:@codingame/monaco-vscode-api@~1.69.21",
-    "vscode-oniguruma": "~1.6.2"
+    "monaco-editor": "~0.34.1",
+    "throttle-debounce": "~5.0.0",
+    "vscode": "file:../",
+    "vscode-oniguruma": "~1.7.0"
```

Comment:
I don't know if we should update it until we upgraded to monaco 0.36, I had related issues in the past

Reply:
Ah, yeah, this is not in line with 0.34